### PR TITLE
Least squares psf

### PR DESCRIPTION
**Pull request by rgiordan**
Add a (pretty slow) least squares fit method to the PSF module.

### CODE REVIEW

File: src/PSF.jl [hunk 1]
```diff
@@ -2,6 +2,8 @@ module PSF
 
 VERSION < v"0.4.0-dev" && using Docile
```
Comment (jeff-regier) [inline src/PSF.jl]: Not needed, we're only supporting 0.4 and 0.5 now.

**Comment by rgiordan** [comment]
Removed Docile stuff.

**Comment by jeff-regier** [comment]
`UndefVarError: NelderMead not defined`

**Comment by rgiordan** [comment]
Aha, Optim's default documentation doesn't match its last tagged version.
The last release, v0.4.4, uses different syntax than the master version
that I am running on my computer.

I'll switch to the obsolete syntax, I suppose.

**Pull request merged**